handlers/error.go

```diff
@@ -50,6 +50,13 @@ const (
 	ErrApplicationFailureCode = "2042"
 	ErrDecodingCode           = "2043"
 	ErrRetrieveUserTokenCode  = "2044"
+	ErrInvalidGenValueCode	  = "2045"
+	ErrUserPreferencesCode	  = "2046"
```

Comment:
In the new PR that was merged, Error Code 2045 and 2046 were used, you might want to start from 2047